app/services/degrees/create_from_hesa.rb

```diff
@@ -39,13 +39,19 @@ def country_specific_attributes(degree, hesa_degree)
 
       # Country code is not always provided, so we have
       # to fallback to institution which is always UK based
```

Comment:
Offhand, I think it might be worth merging the comment on lines 43-45 with the comment on line 40-41, and line 37 could be clarified that this is where we figure out whether the HESA list has the institution or not.

Approved anyway.

Reply:
I think the preceding comment is still true. What we weren't expecting is that we get some UK degrees with a country code and no institution.